src/client/assets/javascripts/app/routes.js

```diff
@@ -5,11 +5,13 @@ import App from './App';
 import ActionEnter from 'components/ActionEnter';
 import AddAction from 'components/AddAction';
 import NotFoundView from 'components/NotFound';
+import Welcome from 'components/Welcome';
 
 export default (
 	<Route path="/" component={App}>
-		<IndexRoute component={AddAction} />
+		<IndexRoute component={Welcome} />
 		<Route path="404" component={NotFoundView} />
+		<Route path="select" component={AddAction} />
```

Comment:
@kodell Oh right. Just pushed up a new version, but the page is crashing after I click a type of work now. I thought maybe I had to edit AddAction.js around line 15 but couldn't get it to work.